easybuild/easyconfigs/b/beagle-lib/beagle-lib-2_1_2-foss-2016a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'beagle-lib'
+version = '2_1_2'
+
+homepage = 'http://code.google.com/p/beagle-lib/'
+description = """
+beagle-lib is a high-performance library that can perform the core
+calculations at the heart of most Bayesian and Maximum Likelihood
+phylogenetics packages.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# https://github.com/beagle-dev/beagle-lib/archive/beagle_release_2_1_2.tar.gz
+source_urls = ['https://github.com/beagle-dev/beagle-lib/archive/']
+sources = ['beagle_release_%(version)s.tar.gz']
+
+dependencies = [('Java', '1.7.0_15', '', True)]
```

Comment:
this is very old, if you want to stick to Java 1.7, please use the latest version (1.7.0_80)